src/status_im/ui/screens/chat/message/message.cljs

```diff
@@ -99,14 +93,16 @@
      [react/view
       (when (:response-to content)
         [quoted-message (:response-to content) outgoing current-public-key])
-      [react/text (cond-> {:style           (style/text-message collapsible? outgoing)}
-                    (and collapsible? (not expanded?))
-                    (assoc :number-of-lines constants/lines-collapse-threshold))
+      [react/text
+       (cond-> {:style (style/text-message collapsible? outgoing)}
+         (and collapsible? (not expanded?))
+         (assoc :number-of-lines constants/lines-collapse-threshold))
        (if-let [render-recipe (:render-recipe content)]
          (chat.utils/render-chunks render-recipe message)
          (:text content))
-       [react/text {:style (style/message-timestamp-placeholder-text outgoing)}
-        (timestamp-with-padding timestamp-str)]]
+       [react/text {:style {:color (if outgoing colors/blue colors/blue-light)}}
+        (str " AAAAA")]]
```

Comment:
could you elaborate ` (str " AAAAA")` ?

Reply:
It's an invisible field to place the timestamp properly, spaces are trimmed so they wouldn't work

Reply:
hm, how is it invisible? it even has a colour

Reply:
@yenda if we are going to such lengths to have the timestamp in place, maybe, a different design for the timestamp is required altogether? they only really position themselves correctly like 50% of the time, often it's just hanging weird.

Reply:
@errorists yeah that would be great I was thinking about asking you that yesterday. The main issue being that it seems impossible to get text wrapping and alignment of timestamp on the right of the last line of text without overlapping